databricks/conftest.py

```diff
@@ -27,3 +42,18 @@ def add_ks(doctest_namespace):
 @pytest.fixture(autouse=True)
 def add_np(doctest_namespace):
     doctest_namespace['np'] = numpy
+
+
+@pytest.fixture(autouse=True)
+def add_path(doctest_namespace):
+    path = tempfile.mkdtemp()
+    atexit.register(lambda: shutil.rmtree(path, ignore_errors=True))
+    doctest_namespace['path'] = path
+
+
+@pytest.fixture(autouse=True)
+def add_db(doctest_namespace):
+    db_name = str(uuid.uuid4()).replace("-", "")
+    utils.default_session().sql("CREATE DATABASE %s" % db_name)
+    atexit.register(lambda: session.sql("DROP DATABASE IF EXISTS %s CASCADE" % db_name))
```

Comment:
`utils.default_session()` instead of `session`?
Seems like we got errors in Python 3.5 Job:

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/travis/build/databricks/koalas/databricks/conftest.py", line 58, in <lambda>
    atexit.register(lambda: session.sql("DROP DATABASE IF EXISTS %s CASCADE" % db_name))
NameError: name 'session' is not defined
```

Reply:
oh nice catch!